modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/edit_file_entry.jsp

```diff
@@ -422,40 +420,35 @@ renderResponse.setTitle(headerTitle);
 									</div>
 								</c:if>
 
-							<%
-							try {
-								boolean localizable = true;
-
-								for (DDMStructure ddmStructure : dlFileEntryType.getDDMStructures()) {
-									com.liferay.dynamic.data.mapping.storage.DDMFormValues ddmFormValues = null;
+								<%
+								try {
+									boolean localizable = true;
 
-									try {
-										DLFileEntryMetadata fileEntryMetadata = DLFileEntryMetadataLocalServiceUtil.getFileEntryMetadata(ddmStructure.getStructureId(), fileVersionId);
+									for (DDMStructure ddmStructure : dlFileEntryType.getDDMStructures()) {
+										com.liferay.dynamic.data.mapping.storage.DDMFormValues ddmFormValues = null;
 
-										ddmFormValues = dlEditFileEntryDisplayContext.getDDMFormValues(fileEntryMetadata.getDDMStorageId());
-									}
-									catch (Exception e) {
-									}
-							%>
+										try {
+											DLFileEntryMetadata fileEntryMetadata = DLFileEntryMetadataLocalServiceUtil.getFileEntryMetadata(ddmStructure.getStructureId(), fileVersionId);
 
-										<c:if test="<%= !dlEditFileEntryDisplayContext.isDDMStructureVisible(ddmStructure) %>">
```

Comment:
Why are we removing this `c:if`s? I can't see why removing them is needed to fix your issue. If it is needed, can you do it in a separate commit explaining the reasoning behind the change?

Reply:
Oh I see what you've one. You're conditionally adding the class. I got confused. I still think separating this in 2 different commits would have helped me to understand why you're doing this.

Reply:
Updated the commits